libsolidity/codegen/ArrayUtils.h

```diff
@@ -67,7 +70,7 @@ class ArrayUtils
 	/// Appends a loop that clears a sequence of storage slots of the given type (excluding end).
 	/// Stack pre: end_ref start_ref
 	/// Stack post: end_ref
-	void clearStorageLoop(Type const& _type) const;
+	void clearStorageLoop(TypePointer const& _type) const;
```

Comment:
I don't know if `const` here is intended.  Does this mean the reference counters are not updated?